test/other/metadce/test_metadce_minimal_O0.jssize

```diff
@@ -1 +1 @@
-19537
+18753
```

Comment:
Why does strict mode improve code size?

Reply:
It looks like the difference comes from the list unexpected library function and runtime methods.    In `STRICT` mode we don't include `library_legacy.py` or `runtime_legacy.py` so those functions don't show up.